.github/workflows/harmony-in-a-box.yml

```diff
@@ -23,24 +23,38 @@ jobs:
         brew update
         brew install docker
         brew install kubectl
-        colima start --cpu 2 --memory 12 --with-kubernetes
 
     - name: Install awslocal
       run: |
         pip install awscli-local
 
-    - name: Print awslocal Version
-      run: |
-        awslocal --version
+    - name: Set up npm dependency caching
+      uses: actions/setup-node@v4
+      with:
+        node-version: 18
+        cache: 'npm'
+        cache-dependency-path: '**/package-lock.json'
 
-    - name: Pull down basic harmony docker images
-      run: bin/pull-harmony-images
+    - name: Create netrc
+      run: bin/create-netrc
+      env:
+        EDL_USER: ${{ secrets.LEO_EDL_USER }}
```

Comment:
I was thinking we have EDL_USER and EDL_PASSWORD secrets already - but not sure where you check that.

Reply:
I kept running into harmony request failures and that made me think if it is because something is messed up with Patrick's EDL credential either with EDL login or access to the collection. In order to control/test it better, I changed it to use my credential and it actually helped.